Rubberduck.Core/UI/CodeExplorer/Commands/ExportCommand.cs

```diff
@@ -8,33 +8,53 @@
 using Rubberduck.VBEditor;
 using Rubberduck.VBEditor.ComManagement;
 using Rubberduck.VBEditor.SafeComWrappers;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
 
 namespace Rubberduck.UI.CodeExplorer.Commands
 {
     public class ExportCommand : CommandBase
     {
-        private static readonly Dictionary<ComponentType, string> ExportableFileExtensions = new Dictionary<ComponentType, string>
+        private static readonly Dictionary<ComponentType, string> VBAExportableFileExtensions = new Dictionary<ComponentType, string>
         {
             { ComponentType.StandardModule, ".bas" },
             { ComponentType.ClassModule, ".cls" },
             { ComponentType.Document, ".cls" },
-            { ComponentType.UserForm, ".frm" }
+            { ComponentType.UserForm, ".frm" }            
+        };
+
+        private static readonly Dictionary<ComponentType, string> VB6ExportableFileExtensions = new Dictionary<ComponentType, string>
+        {
+            { ComponentType.StandardModule, ".bas" },
+            { ComponentType.ClassModule, ".cls" },
+            { ComponentType.VBForm, ".frm" },
+            { ComponentType.MDIForm, ".frm" },
+            { ComponentType.UserControl, ".ctl" },
+            { ComponentType.DocObject, ".dob" },
+            { ComponentType.ActiveXDesigner, ".dsr" },
+            { ComponentType.PropPage, ".pag" },
+            { ComponentType.ResFile, ".res" },            
         };
 
         private readonly IFileSystemBrowserFactory _dialogFactory;
+        private readonly Dictionary<ComponentType, string> _ExportableFileExtensions;
```

Comment:
the other `private readonly` uses lower camel case

Reply:
bah, switchfail between work coding standards and standard standards :-)

Reply:
fixed